### PR TITLE
Chaplains now enjoy eating the only divine food

### PR DESCRIPTION
**Pull request by Memedoktor**
## About The Pull Request
The Chaplain now gets a nice little new moodlets from eating holymelons
It is a a +5 mood boost for 3 minutes, compared to the favorite foods getting +5 for 4 minutes.

## Why It's Good For The Game
Gives the chaplains reasons to interact with the other service jobs.

## Changelog
:cl: 
tweak: adds positive moodlet for chaplain eating holymelon

/:cl:

### CODE REVIEW

File: code/modules/hydroponics/grown/melon.dm [hunk 1]
```diff
@@ -77,6 +77,17 @@
 	qdel(src)
 	new /obj/effect/decal/cleanable/ash(drop_location())
 
+/obj/item/reagent_containers/food/snacks/grown/holymelon/checkLiked(fraction, mob/M)    //chaplains sure love holymelons
```
Comment (ATH1909) [inline code/modules/hydroponics/grown/melon.dm:80]: you probably need a ..() somewhere in this proc

File: code/modules/hydroponics/grown/melon.dm [hunk 2]
```diff
@@ -77,6 +77,17 @@
...
+    if(last_check_time + 50 < world.time)
+        if(ishuman(M))
+            var/mob/living/carbon/human/H = M
+            if(H.mind?.holy_role && !HAS_TRAIT(H, TRAIT_AGEUSIA))
+                to_chat(H,"<span class='notice'>Truly a piece of heaven</span>")
```
Comment (ATH1909) [inline code/modules/hydroponics/grown/melon.dm:85]:
```suggestion
                to_chat(H,"<span class='notice'>Truly, a piece of heaven!</span>")
```

File: code/datums/mood_events/generic_positive_events.dm [hunk 3]
```diff
@@ -198,3 +198,8 @@
 /datum/mood_event/confident_mane
 	description = "<span class='nicegreen'>I'm feeling confident with a head full of hair.</span>\n"
 	mood_change = 2
+
+/datum/mood_event/holy_consumption
+    description = "<span class='nicegreen'>Truly that was the food of the Divine!.</span>\n"
```
Comment (ATH1909) [inline code/datums/mood_events/generic_positive_events.dm:203]:
```suggestion
    description = "<span class='nicegreen'>Truly, that was the food of the Divine!.</span>\n"
```

Comment (Jared-Fogle) [reply code/datums/mood_events/generic_positive_events.dm:203]: you did a spell check comment and didnt notice the `!.`

Comment (ATH1909) [reply code/datums/mood_events/generic_positive_events.dm:203]: whoopsie doopsie

**Comment by ATH1909** [comment]
make this also apply for eating ambrosia gaia

**Comment by Memedoktor** [comment]
@ATH1909  thanks for the spelling fixes i missed those in my effort to get the thing to actually work given its my first pr.

**Comment by Learner10** [comment]
Don't forget Angel Cake as well.

**Comment by Memedoktor** [comment]
> Don't forget Angel Cake as well.

I'll do that in a follow up, i wanna leave this one as unborked as possible.

**Comment by Mycah142** [comment]
Gives Chaplains motivation to leave holy water tanks by service departments.

Finally my dumb habits can be justified.

**Comment by Memedoktor** [comment]
not stale, in need of review

**Review by Rohesie — CHANGES_REQUESTED** [review]
PR looks fine, just do the requested changes and it's otherwise approved.

File: code/datums/mood_events/generic_positive_events.dm [hunk 4]
```diff
@@ -198,3 +198,8 @@
 /datum/mood_event/confident_mane
 	description = "<span class='nicegreen'>I'm feeling confident with a head full of hair.</span>\n"
 	mood_change = 2
+
+/datum/mood_event/holy_consumption
+    description = "<span class='nicegreen'>Truly, that was the food of the Divine!</span>\n"
+    mood_change = 5
+    timeout = 3 MINUTES
```
[the remark this answers is not in the document]
Comment (Memedoktor) [reply code/datums/mood_events/generic_positive_events.dm]: Thanks for the advice the person who helped me i guess hadn't noticed. They **should** be good now.

**Comment by Memedoktor** [comment]
alright folks 3-5th addendum the charm

**Comment by Rohesie** [comment]
Are you web-editing this PR?

**Comment by Memedoktor** [comment]
@Rohesie Yeah rave when i was fixing a couple of the initial errors said that it was fine.

**Comment by Rohesie** [comment]
Avoid web-editing, it's prone to errors. Try using a proper git client in the future.

**Pull request merged**